rpc/ub/plot.sh

```diff
@@ -116,7 +116,7 @@ get_val() {
     local KEY="$1"
     local OPTS="$2"
 
-    echo "$OPTS" | tr , \\n | sed -n "s/^$KEY=//p"
+    echo $OPTS | tr , \\n | sed -n "s/^$KEY=//p"
```

Comment:
@rkothiya, Need to ignore the double quote warning here.

Reply:
done